pkgs/applications/video/mirakurun/mirakurun.1

```diff
@@ -0,0 +1,44 @@
+.Dd $Mdocdate$
```

Comment:
We normally don't write man pages but use the nixos manual instead

Reply:
As noted in this document, the primary way of using Mirakurun is through the NixOS module. So the main document is in the options section of the NixOS manual, and this man page isn't something that I meant to prominently feature in the manual.

Additionally, this is  mainly written for people who try to run the command directly and get confused. For that audience, it's not very clear at first that this is an Nix-provided command instead of something provided by upstream and I think it's natural to look in the man pages.